src/main/java/net/dv8tion/jda/bot/sharding/DefaultShardManagerBuilder.java

```diff
@@ -0,0 +1,573 @@
+     * <br>When the Java program closes shutdown hooks are run. This is used as a last-second cleanup
+     * attempt by JDA to properly close connections.
+     *
+     * <p>Default: <b>true (enabled)</b>
+     *
+     * @param  enable
+     *         True (default) - use shutdown hook to clean up JDA if the Java program is closed.
+     *
+     * @return Return the {@link net.dv8tion.jda.bot.sharding.DefaultShardManagerBuilder DefaultShardManagerBuilder} instance. Useful for chaining.
+     */
+    public DefaultShardManagerBuilder setEnableShutdownHook(final boolean enable)
+    {
+        this.enableShutdownHook = enable;
+        return this;
+    }
+
+    /**
+     * Changes the internally used EventManager.
+     * <br>There are 2 provided Implementations:
+     * <ul>
+     *     <li>{@link net.dv8tion.jda.core.hooks.InterfacedEventManager InterfacedEventManager} which uses the Interface
+     *     {@link net.dv8tion.jda.core.hooks.EventListener EventListener} (tip: use the {@link net.dv8tion.jda.core.hooks.ListenerAdapter ListenerAdapter}).
+     *     <br>This is the default EventManager.</li>
+     *
+     *     <li>{@link net.dv8tion.jda.core.hooks.AnnotatedEventManager AnnotatedEventManager} which uses the Annotation
+     *         {@link net.dv8tion.jda.core.hooks.SubscribeEvent @SubscribeEvent} to mark the methods that listen for events.</li>
+     * </ul>
+     * <br>You can also create your own EventManager (See {@link net.dv8tion.jda.core.hooks.IEventManager}).
+     *
+     * @param  manager
+     *         The new {@link net.dv8tion.jda.core.hooks.IEventManager} to use.
+     *
+     * @return The {@link net.dv8tion.jda.bot.sharding.DefaultShardManagerBuilder DefaultShardManagerBuilder} instance. Useful for chaining.
+     */
+    public DefaultShardManagerBuilder setEventManager(final IEventManager manager)
+    {
+        this.eventManager = manager;
+        return this;
+    }
+
+    /**
+     * Sets the {@link net.dv8tion.jda.core.entities.Game Game} for our session.
+     * <br>This value can be changed at any time in the {@link net.dv8tion.jda.core.managers.Presence Presence} from a JDA instance.
+     *
+     * <p><b>Hint:</b> You can create a {@link net.dv8tion.jda.core.entities.Game Game} object using
+     * {@link net.dv8tion.jda.core.entities.Game#of(String) Game.of(String)} or
+     * {@link net.dv8tion.jda.core.entities.Game#of(String, String) Game.of(String, String)}.
+     *
+     * @param  game
+     *         An instance of {@link net.dv8tion.jda.core.entities.Game Game} (null allowed)
+     *
+     * @return The {@link net.dv8tion.jda.bot.sharding.DefaultShardManagerBuilder DefaultShardManagerBuilder} instance. Useful for chaining.
+     *
+     * @see    net.dv8tion.jda.core.managers.Presence#setGame(Game)
+     */
+    public DefaultShardManagerBuilder setGame(final Game game)
+    {
+        this.game = game;
+        return this;
+    }
```

Comment:
This method should be grouped together with `setIdle` and `setStatus` for better navigation.